pkg/cli/image/strategy/onerror.go

```diff
@@ -126,9 +126,9 @@ func alternativeImageSources(imageRef reference.DockerImageReference, icspList [
 // readICSPsFromFileFunc is used for testing to be able to inject ICSP data
 type readICSPsFromFileFunc func(string) ([]operatorv1alpha1.ImageContentSourcePolicy, error)
 
-// readICSPsFromFile appends to list of alternative image sources from ICSP file
+// ReadICSPsFromFile appends to list of alternative image sources from ICSP file
 // returns error if no icsp object decoded from file data
-func readICSPsFromFile(icspFile string) ([]operatorv1alpha1.ImageContentSourcePolicy, error) {
+func ReadICSPsFromFile(icspFile string) ([]operatorv1alpha1.ImageContentSourcePolicy, error) {
```

Comment:
I'd suggest copying this method over to your command, since when we introduce `ImageDigestMirrorSet`, this function will need to know how to read both of those.

Reply:
Also, I assume a followup to introduce `ImageDigestMirrorSet` here will be soon opened.